Assets/Scripts/AsteroidCollisionHealthEffect.cs

```diff
@@ -2,9 +2,15 @@
 
 public class AsteroidCollisionHealthEffect : HealthEffectBehavior
 {
-    public override int Strength => damage;
-    public override HealthEffect Effect => HealthEffect.Damage;
+    public override int ComputeStrength(Collision collision = null) => collision == null
+        ? 0
+        : Mathf.RoundToInt(Mathf.Clamp(collision.impulse.magnitude / Time.fixedDeltaTime * (damageMultiplier / 1000),
```

Comment:
I feel like this would read more intuitively if it was `collision.impulse.magnitude / Time.fixedDeltaTime / 1000f * damageMultiplier`